x-pack/osquerybeat/internal/action/action.go

```diff
@@ -121,3 +140,31 @@ func parseECSMapping(m map[string]interface{}) (ecsm ecs.Mapping, err error) {
 	}
 	return ecsm, err
 }
+
+func convertToInt64(i interface{}) (int64, error) {
+	switch v := i.(type) {
```

Comment:
Do we expect this many different types to be in the map?

Reply:
Not really, effectively it's parsed from JSON and ends up as float64, but don't want to rely on this precondition in case if something changes later on in the processing pipeline